R/style_guides.R

```diff
@@ -142,8 +142,8 @@ tidyverse_style <- function(scope = "tokens",
 
   indention_modifier <-
     c(
-      update_indention_ref_fun_dec,
-      NULL
+      if (scope >= "indention") update_indention_ref_fun_dec,
+      identity
```

Comment:
I'm not in love with the use of `identity` here, this sure doesn't help performance. Maybe allow `NULL` and filter out later with `compact()`? Need to use `lst()` instead of `c()`.

The name for the list elements will be the full quoted expression, maybe change l. 138 above to `wrap_if_else_multi_line_in_curly = if (strict) wrap_if_else_multi_line_in_curly else identity` and likewise in other places?

Reply:
Ok, I like the idea with `purrr::compact()`, `lst()` and naming the arguments here and in other places.